torchao/quantization/README.md

```diff
@@ -74,7 +74,7 @@ from torchao.quantization.quant_primitives import MappingType, ZeroPointDomain
 from torchao.dtypes import to_affine_quantized
```

Comment:
The main README still has quantize listed, mind doing a comprehensive code search?

Reply:
done